src/databricks/labs/remorph/queries/tables/reconcile/main.sql

```diff
@@ -0,0 +1,18 @@
+CREATE OR REPLACE TABLE main (
+    recon_table_id BIGINT NOT NULL,
+    recon_id STRING NOT NULL,
+    source_type STRING NOT NULL,
+    source_table STRUCT<
+                         catalog: STRING NOT NULL,
```

Comment:
Sine we allow source catalog as null in our input .. here not null should be removed.